File: Modul4/Soal 5/PRAK305-2210817210010-Eric Nanda Ferdian.c
#include <stdio.h>

int main()
{
    int sec,hari,jam,menit;
    scanf("%d",&sec);
    hari = sec / 86000;
    jam = sec / 3600;
    sec = sec -(3600*jam);
    menit = sec / 60;
    sec = sec - (60*menit);

    if(jam>23)
    {
        jam = jam - 24;
        printf("%d hari %02d : %02d : %02d",hari,jam,menit,sec);
    }
    else if(jam<24)
    {
        printf("%02d : %02d : %02d",jam,menit,sec);
    }
}

File: Modul4/Soal 5/PRAK305-2210817210010-Eric Nanda Ferdian.py
sec = int(input(''))
hari = int(sec) / 86000
jam = int(sec) / 3600
sec = int(sec) % 3600
(menit) = int(sec) / 60
sec = int(sec) % 60
if jam > 23:
    jam = int(jam) - 24
    print('%s hari %s : %s : %s' % (int(hari),int(jam),int(menit),sec))
elif jam < 24:
    print('%s : %s : %s' % (int(jam),int(menit),sec))